Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node
+FROM node:stretch
```

Comment:
stretch is 74mo bigger than latest, Is there a particular reason for switching ?